app/registration/controllers/patientCommonController.js

```diff
@@ -32,13 +32,14 @@ angular.module('registration')
                                 $scope.patient.identifiers.push({identifierType: value,
                                     identifier: null, preferred: false,
                                     location: localStorageService.cookie.get("emr.location").uuid,
-                                    fieldName : fieldName});
+                                    fieldName : fieldName,
+                                    uimask: "(99999999/99/99999)",
```

Comment:
Yes. Thank you for the feedback, let me work on improving that and increment the PR